map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -196,6 +196,24 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
              * locks us into just that area
              */
             setLatLngBoundsForCameraTarget(mapDisplayBounds)
+
+            /**
+             * Marker.showInfoWindow() method is used (@see MapViewModel.retrieveMarkerByObjectId)
+             * to bring the selected marker to front (markers are sometime overlapped by other
```

Comment:
I think the inline link style `(see [MapViewModel.retrieveMarkerByObjectId])` would be better here. Also, how about `markers sometimes overlap` for the parenthetical on the next line?

Reply:
Agreed. Updated the documentation.